### DISCUSSION TITLE
Is it possible to use local file storage instead of S3

### DISCUSSION THREAD

**Discussion by ShayPunter** [outline/outline#1651] [Self Hosting] [answered]
Hello! 

Just wondering if it's possible to use local file storage instead of S3. I have a self-installed version and would prefer if I have all my files locally stored as opposed to on the cloud, this is for cost reasons and privacy reasons.

**Comment by tommoor** [comment, marked as answer]
Yes, you can run any S3-compatible service. If you want to run it locally a lot of users have had success using a [Minio](https://github.com/minio/minio) container.
